src/Places/DataAccess.ts

```diff
@@ -134,15 +135,26 @@ export async function getPlaceMedia(id: string): Promise<Medium[]> {
 }
 
 async function getFromApi(toBeFetchedFromAPI: string[]): Promise<any> {
-	const apiResponse = await StApi.get('places?' + stringify({
-		ids: toBeFetchedFromAPI.join('|')
+	const CHUNK_SIZE: number = 50;
+	const idsChunks = splitArrayToChunks(toBeFetchedFromAPI, CHUNK_SIZE);
+	const apiResponsesForChunks: ApiResponse[] = await Promise.all(idsChunks.map((idsChunk: string[]) => {
+		return StApi.get('places?' + stringify({
+			ids: idsChunk.join('|')
+		}));
 	}));
 
-	if (!apiResponse.data.hasOwnProperty('places')) {
-		throw new Error('Wrong API response');
-	}
+	apiResponsesForChunks.forEach((apiResponse: ApiResponse) => {
+		if (!apiResponse.data.hasOwnProperty('places')) {
+			throw new Error('Wrong API response');
+		}
+	});
 
-	return apiResponse.data.places;
+	return apiResponsesForChunks.reduce((acc: any[], apiResponse: ApiResponse) => {
```

Comment:
this is a customized programmed a flatten method.

I would extract the logic into own function. (https://stackoverflow.com/questions/10865025/merge-flatten-an-array-of-arrays-in-javascript)

Second, you have already done a check for palces key, why it is here duplicated?

The overall coulkd be (included your previous block with forEach done as follows:

```ts
val chunkedPlaces = apiResponses.map(() => {
		if (!apiResponse.data.hasOwnProperty('places')) {
				throw new Error('Wrong API response');
		}
		return apiResponse.data.places;
});
return flatten(chunkedPlaces);
```